src/pages/VFP01_CropFace.page

```diff
@@ -0,0 +1,75 @@
+<apex:page standardController="Account" extensions="CropFaceController">
+    <script>
+    var latestImageViewed;
+    var eventMethod = window.addEventListener ? "addEventListener" : "attachEvent";
+    var eventer = window[eventMethod];
+    var messageEvent = eventMethod == "attachEvent" ? "onmessage" : "message";
+    eventer(messageEvent, function(e) {
+        if (e.origin === 'https://app.sharinpix.com') {
+            var data = e.data;
+            switch (data.name) {
+                case 'viewer-image-viewed':
+                    latestImageViewed = data.payload.image.public_id;
+                    console.log(latestImageViewed);
+                    break;
+                case 'viewer-closed':
+                    latestImageViewed = null;
+            }
+        }
+    }, false);
+
+    //CROPPING
+    function cropFace(){
+        var height = document.getElementById("heightInput").value;
+        var width = document.getElementById("widthInput").value;
+        var crop = document.getElementById("cropInput").value;
+        var gravity = document.getElementById("gravityInput").value;
+        var radius = document.getElementById("radiusInput").value;
+
+        Visualforce.remoting.Manager.invokeAction(
+            '{!$RemoteAction.CropFaceController.cropFace}',
+            latestImageViewed,
+            width,//width
+            height,//height
+            crop,//crop option
+            gravity,//gravity
+            radius, //radius
+            function(result, event){
+                if (event.status) {
+                   
+                    var WindowObjectReference = window.open(
```

Comment:
remove WindowObjectReference
(should have been windowObjectReference anyway => camelCase)